featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -569,12 +576,15 @@ def parallel_calculate_chunks(chunks, features, approximate, training_window,
         pickled_feats = cloudpickle.dumps(features)
         _saved_features = client.scatter(pickled_feats)
         client.replicate([_es, _saved_features])
+        num_scattered_workers = len(client.who_has([Future(es_token)]).get(es_token, []))
+        num_workers = len(client.scheduler_info()['workers'].values())
+
+        scatter_warning(num_scattered_workers, num_workers)
         if verbose:
             end = time.time()
-            scatter_time = end - start
-            scatter_string = "EntitySet scattered to workers in {:.3f} seconds"
-            print(scatter_string.format(scatter_time))
-
+            scatter_time = round(end - start)
```

Comment:
instead of rounding up `round` rounds to the nearest integer.  We could use a function that rounds up or take the min of the rounded number and 1

Reply:
What is the motivation for wanting to tell the user it took 1 second if it actually took 0.001 seconds

Reply:
I thought saying something took 0 seconds seem strange to the user since clearly it took at least some time to do, but it's probably fine so let's just leave it as is.